nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -61,6 +61,8 @@ class BtActionNode : public BT::ActionNodeBase
     server_timeout_ =
       config().blackboard->template get<std::chrono::milliseconds>("server_timeout");
     getInput<std::chrono::milliseconds>("server_timeout", server_timeout_);
+    bt_wait_for_service_timeout_ =
+      config().blackboard->template get<std::chrono::milliseconds>("bt_wait_for_service_timeout_");
```

Comment:
```suggestion
      config().blackboard->template get<std::chrono::milliseconds>("bt_wait_for_service_timeout");
```

Reply:
Also, for all, remove the `bt_` prefix. This is in the BT navigator node, its pretty clear in context at this is for BTs